tools/OpenConsole.psm1

```diff
@@ -363,16 +363,91 @@ function Invoke-ClangFormat {
     }
 }
 
+function Invoke-StripBOM {

```

Comment:
The original author of this module set a poor precedent for function names :/

Reply:
Since this isn't exported, it can be named whatever we want. Like, `StripBOM`

Reply:
okay